testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/security/credentialreference/CredentialStoreServerSetupTask.java

```diff
@@ -58,7 +57,8 @@ public void tearDown(ManagementClient managementClient, String s) throws Excepti
 
     private void createCredentialStore(final ModelControllerClient client) throws IOException {
         final ModelNode addOperation = Operations.createAddOperation(CREDENTIAL_STORE_ADDRESS.toModelNode());
-        addOperation.get(URI).set("cr-store://test/store001.jceks?create=true;keyStoreType=JCEKS;modifiable=true");
```

Comment:
yes, JCEKS is default KeyStore type for KeyStoreCredentialStore
yes, location might be set, but it is inherited from name of resource (in this case it doesn't matter)